### ISSUE TITLE
[Bug] Handle new line character and Enter key properly to perform usual operations

### ISSUE THREAD

**Issue by sujal-goswami** [sujal-goswami/FlashType#6]
When a text string contains a `;` character, the text cursor moves to the next line and begins to write. The `Enter key` also not functioning normally.

Check out this function for references. 

<img width="399" alt="image" src="https://github.com/sujal-goswami/FlashType/assets/107000806/2e28dba8-8b90-4606-b933-97273290d2f8">

**Comment by Adishumla** [comment]
Hello, Sujal!
I would like to fix this issue :)

**Comment by sujal-goswami** [comment]
Of course! go ahead and follow [CONTRIBUTING.md](https://github.com/sujal-goswami/FlashType/blob/main/CONTRIBUTING.md).
